test/functional/interface_rest.py

```diff
@@ -312,6 +312,16 @@ def run_test(self):
                             if 'coinbase' not in tx['vin'][0]}
         assert_equal(non_coinbase_txs, set(txs))
 
+        # Verify that the non-coinbase tx has "prevout" field set
+        for tx_obj in json_obj["tx"]:
+            for vin in tx_obj["vin"]:
+                if "coinbase" not in vin:
+                    assert "prevout" in vin
+                    prevout = vin["prevout"]
+                    assert_equal(prevout["generated"], False)
```

Comment:
Given that `prevout` variable is used just once, it may be shorter and actually easier to read:

```suggestion
                    assert_equal(vin["prevout"]["generated"], False)
```